packages/ethereum/sdk/src/order/fill-order/seaport-utils/order.ts

```diff
@@ -96,7 +96,9 @@ export const mapInputItemToOfferItem = (item: CreateInputItem): OfferItem => {
 				itemType: item.itemType,
 				token: item.token,
 				identifierOrCriteria: item.identifier,
+				// @ts-ignore
 				startAmount: item.amount,
+				// @ts-ignore
```

Comment:
Let's mark this ignore files with @todo tag of jsdoc. We should keep them in mind and fix soon